SpatialGDK/Source/SpatialGDKServices/Public/SpatialPackageService.h

```diff
@@ -0,0 +1,38 @@
+﻿// Copyright (c) Improbable Worlds Ltd, All Rights Reserved
+
+#pragma once
+
+#include "CoreMinimal.h"
+#include "SpatialGDKServicesConstants.h"
+
+DECLARE_LOG_CATEGORY_EXTERN(LogSpatialPackageService, Log, All);
+
+// Represents a package that can be downloaded via the package service.
+// This currently assumes that the package will extract into at least an executable with the same name as Type.
+struct FSpatialPackageDescriptor
+{
+	const FString Type;
```

Comment:
Names are a bit confusing IMO.
Given the rest of the code, I would replace "Type" by "Name", and "Name" by "BuildVariant", or "Platform"

Reply:
Yes, that's true but I decided to use the terminology used by the `package_client`, and there it's `Type`, `Name` and `Version`. Not married to this as this is our own internal interface. I'm just not sure how consistently the packages are named themselves, so it might be confusing in any case.

Reply:
I think we should maintain the vars as package_client specifies.